src/Lykke.Sdk/Controllers/IsAliveController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Lykke.Common;
+using Lykke.Common.Api.Contract.Responses;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using System.Diagnostics;
+using System.Net;
+
+namespace Lykke.Sdk.Controllers
+{
+    // NOTE: See https://lykkex.atlassian.net/wiki/spaces/LKEWALLET/pages/35755585/Add+your+app+to+Monitoring
+    /// <summary>
+    /// Common controller for app health monitoring.
+    /// </summary>
+    [Route("api/[controller]")]
+    public class IsAliveController : Controller
+    {
+        /// <summary>
+        /// Checks service is alive
+        /// </summary>
+        [HttpGet]
+        [SwaggerOperation("IsAlive")]
+        [ProducesResponseType(typeof(IsAliveResponse), (int)HttpStatusCode.OK)]
+        public IActionResult Get()
+        {
+            // NOTE: Feel free to extend IsAliveResponse, to display job-specific indicators
```

Comment:
how can this be extended with indicators?

Reply:
In case of HFT the IHealthService is being used to extend with additional health indicators.

https://github.com/LykkeCity/Lykke.Service.HFT/blob/test/src/Lykke.Service.HFT/Controllers/IsAliveController.cs

Reply:
But there are no inidcators in the response. Will be better to add retrieve indicators from the health service and return them